File: Python/Warmup-2/string_match.py
def string_match(a, b):
    if len(a) <= len(b):
        str = a
    else:
        str = b

    count = 0
    for i in range(len(str) - 1):
        if a[i] == b[i] and a[i + 1] == b[i + 1]:
            count += 1
            continue
    return count

File: Java/Warmup-2/stringMatch.java
public int stringMatch(String a, String b) {
    String str = "";
    if (a.length() <= b.length()) {
        str = a;
    } else {
        str = b;
    }


    int count = 0;
    for(int i = 0; i < str.length() - 1; i++) {
        if(a.charAt(i) == b.charAt(i) && a.charAt(i + 1) == b.charAt(i + 1)) {
            count++;
        }
    }
    return count;
}
